### PR TITLE
[PT] Update section on mocking

### PR DESCRIPTION
**Pull request by anderkonzen**
Related to #680

### CODE REVIEW

**Comment by ybur-yug** [comment]
If someone finalizes the OK on proofreading I'll merge this :)

Just wanna be absolutely sure because I obviously don't know the language so the more eyes the better :P

this one seems simple enough but I just wanna be sure of grammar in the last paragraph where its not trivial changes.

**Comment by anderkonzen** [comment]
Sure, no problem!

**Comment by doomspork** [comment]
@pragmaticivan / @capaci — would either of you mind looking this over when you have a moment? 😀

File: pt/lessons/basics/testing.md [hunk 1]
```diff
@@ -101,6 +102,8 @@ end
...
+A resposta simples para mocking no Elixir é: não faça isso. Você pode instintivamente querer utilizar mocks, porém eles são altamente desaconselhados na comunidade Elixir por uma boa razão.
+
+Para uma discussão mais longa, temos este [excelente artigo](http://blog.plataformatec.com.br/2015/10/mocks-and-explicit-contracts/). O ponto principal é que ao invés de mockar dependências para testar (mock como *verbo*), existem muitas vantagens em explicitamente definir interfaces (comportamentos) para código fora da aplicação e usar implementações mockadas (mock como *nome*) no nosso código cliente para testar.
 
-Resista à tentação.
+Para alternar entre as implementações no código da aplicação, a maneira preferida é passar o módulo como argumento e usar um valor padrão. Se isto não funcionar, use o mecanismo de configuração embutido. Para criar estas implementações mockadas, você não precisa uma biblioteca especial de mocking, apenas comportamentos e callbacks.
```
Comment (pragmaticivan) [inline pt/lessons/basics/testing.md]: `você não precisa de uma biblioteca`

Comment (anderkonzen) [inline pt/lessons/basics/testing.md]: Done! Thanks for reviewing this!

**Comment by ybur-yug** [comment]
@pragmaticivan if you approve the review I'll get this one squashed & merged 👍

**Pull request merged**